homeassistant/components/mqtt/util.py

```diff
@@ -13,9 +17,17 @@
     ATTR_TOPIC,
     DEFAULT_QOS,
     DEFAULT_RETAIN,
+    DOMAIN,
 )
 
 
+def mqtt_config_entry_enabled(hass: HomeAssistant) -> bool | None:
+    """Return true when the MQTT config entry is enabled."""
+    if not bool(hass.config_entries.async_entries(DOMAIN)):
+        return None
```

Comment:
It seems unnecessary to make this distinction, see comments above.

Reply:
The state of none existing or disabled is differently handled